frontend/package.json

```diff
@@ -1,18 +1,19 @@
 {
     "name": "findit",
-    "version": "0.1.0",
+    "version": "1.0.0",
     "private": true,
     "dependencies": {
+        "@cthit/react-digit-components": "^1.0.1",
         "font-awesome": "^4.7.0",
-        "@cthit/react-digit-components": "^1.0.0-beta.55",
         "react": "^16.12.0",
-        "react-redux": "^7.1.3",
+        "react-redux": "^7.2.0",
```

Comment:
Remove react-redux.

Reply:
you only need to copy these:
https://github.com/cthit/react-digit-components/blob/master/package.json#L23